### ISSUE TITLE
'Adventure' should be relocated to avoid class loader conflicts

### ISSUE THREAD

**Issue by lokka30** [Archy-X/AureliumSkills#137]
### Server software and version

Paper 1.19.2

### Expected behavior

Adventure is relocated using the Maven Shade Plugin, not causing possible class loader conflicts with other plugins.

### Actual behavior

```
[00:08:33] [Server thread/WARN]: [OtherPluginUsingAdventure] Loaded class net.kyori.adventure.text.Component from AureliumSkills vBeta 1.3.9 which is not a depend or softdepend of this plugin.
```

AureliumSkills does not relocate Adventure, causing potential class loader conflicts with other plugins.

### How to reproduce

Load another plugin which uses Adventure, such as [LevelledMobs](https://www.spigotmc.org/resources/levelledmobs.74304/), alongside AureliumSkills. Note that the 'other plugin' may or may not *shade* Adventure, if the other plugin shades Adventure without relocating it then that's an issue with that plugin. 

### Additional information

Aurelium Skills Beta v1.3.9, latest.

### Agreements

- [X] I am running the latest version of Aurelium Skills
- [X] I am not using an unsupported server software (https://wiki.aurelium.dev/skills/other/incompatibilities)
- [X] My version of Minecraft is supported by Aurelium Skills

**Comment by Archy-X** [comment]
I intended for adventure to be relocated in Slate (the dependency where it is actually used) but it seems like the build.gradle for that has issues, maybe you could take a look and see if you know the issue? I'm not an expert at Gradle so I'm not sure. (https://github.com/Archy-X/AureliumSkills/issues/134)

I have relocated adventure in Aurelium Skills for now: https://github.com/Archy-X/AureliumSkills/commit/62c68af4df3b1ea1f92c4571ef691acc8639891f

**Comment by lokka30** [comment]
Awesome! Thank you for the very prompt fix. I'll close this issue since you fixed it in 62c68af.

I wish I could assist you with Gradle, unfortunately I am terrible at using it, to the point I spent a lot of effort to use Maven instead of Gradle when I was wrote some test Kotlin plugins. 😄
